src/startup/mapping.jsx

```diff
@@ -5,7 +5,7 @@ import { routes } from 'startup/routes'
 
 export const Mapping = () => <>
   {
-    routes.map(route => <Route exact path={route.path} key={route.path}>
+    routes.map(route => <Route path={route.path} key={route.path} exact>
```

Comment:
Was this change really necessary? :smile:

Reply:
Done